minigotchi-ESP32/display.cpp

```diff
@@ -44,10 +46,37 @@ void Display::startScreen() {
       adafruit_display->setTextColor(WHITE);
     } else if (Config::screen == "CYD") {   // Check if the screen configuration is set to "CYD" and execute the corresponding code
       tft.begin();  // Initialize TFT_eSPI library
+      delay(100);
       tft.setRotation(1);  // Set display rotation if needed
+      delay(100);
       tft.fillScreen(TFT_BLACK);  // Fill screen with black color
+      delay(100);
       tft.setTextColor(TFT_WHITE);  // Set text color to white
+      delay(100);
       tft.setTextSize(2);  // Set text size
+      delay(100);
+    } else if (Config::screen == "TTGO_T_DISPLAY") {  // New condition for TTGO_T_DISPLAY
```

Comment:
I already have this display defined, when we merge this we can address the conflicts and stuff

Reply:
Okay, noted.